pkg/shipclient/channel.go

```diff
@@ -121,6 +131,82 @@ func ArchiveChannel(appID string, channelID string) error {
 	return nil
 }
 
-func GetChannel(appID string, channelID string) (interface{}, []interface{}, error) {
-	return nil, nil, nil
+var getShipChannel = `
+  query getChannel($channelId: ID!) {
+    getChannel(channelId: $channelId) {
+      id
+      appId
+      name
+      description
+      channelIcon
+      currentVersion
+      currentReleaseDate
+      installInstructions
+      currentSpec
+      numReleases
+      adoptionRate {
+        releaseId
+        semver
+        count
+        percent
+        totalOnChannel
+      }
+      customers {
+        id
+        name
+        avatar
+        actions {
+          shipApplyDocker
+        }
+        installationId
+        shipInstallStatus {
+          status
+          updatedAt
+        }
+      }
+      githubRef {
+        owner
+        repoFullName
+        branch
+        path
+      }
+      extraLintRules
+      created
+      updated
+      isDefault
+      isArchived
+      releases {
```

Comment:
same as above, do for fetching channels, do we need to get every release that was published to the channel?